eng/Packages.Data.props

```diff
@@ -27,6 +27,7 @@
     <PackageReference Update="Microsoft.ServiceFabric" Version="6.4.624" />
```

Comment:
This file should not be edited, please revert the change here.

Reply:
Change reverted.